packages/universal-components/src/FormLabel/FormLabel.js

```diff
@@ -36,18 +37,22 @@ export default function FormLabel({
   filled,
   disabled,
   inlineLabel,
-  style,
+  labelContainerStyle,
+  labelTextStyle,
 }: Props) {
   return (
     <View
-      style={[inlineLabel ? styles.inlineFormLabel : styles.formLabel, style]}
+      style={[
+        inlineLabel ? styles.inlineFormLabel : styles.formLabel,
+        labelContainerStyle,
+      ]}
     >
       {required && (
         <Asteriks filled={filled} disabled={disabled}>
           *{' '}
         </Asteriks>
       )}
-      <Text style={styles.labelText}>{children}</Text>
+      <Text style={[labelTextStyle, styles.labelText]}>{children}</Text>
```

Comment:
```suggestion
      <Text style={[styles.labelText, labelTextStyle]}>{children}</Text>
```
This means it will be possible to override the style.